package.json

```diff
@@ -54,6 +54,7 @@
   "scripts": {
     "prepublish": "gulp prepublish",
     "pretest": "gulp pretest",
+    "prestart": "webdriver-manager start > /dev/null 2>&1 &",
```

Comment:
Sorry for the late response! Yep, I think child processes are the way to go. I hate adding more layers to this but it's probably best handled in the Gulpfile.